Objects/longobject.c

```diff
@@ -4243,7 +4243,7 @@ long_invmod(PyLongObject *a, PyLongObject *n)
 
     Py_DECREF(c);
     Py_DECREF(n);
-    if (long_compare(a, _PyLong_One)) {
+    if (long_compare(a, (PyObject *)_PyLong_One)) {
```

Comment:
This is wrong, it should be `PyLongObject`.
https://github.com/python/cpython/pull/13761 is the proper fix.